### PR TITLE
The Template for New Pull Requests and New Issues

### LINKED ISSUE #125: Manually Create a Single Issue Template for Gatorgrouper Repository
Reference: https://help.github.com/articles/manually-creating-a-single-issue-template-for-your-repository/

### LINKED ISSUE #127: Create a Pull Request Template for Gatorgrouper Repository
Since teams are going to create a lot of pull requests, I think we need a pull request template to make sure it is easy to differentiate their feature and purpose.
Github does support that: https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/

We probably need to do this ASAP.

### PR DESCRIPTION
**Pull request by Lancasterwu**
# Description of Pull Request

This Pull Request provided the Template for Pull Requests and Issues

With these features, the team work related to this project can more efficient.

Fixes #125 
Fixes #127 

## Type of change

Please descirbe the pull request as one of the following: 

- [ ] Bug fix
- [ ] Breaking change
- [x] New feature
- [ ] Documentation update
- [x] Other (Please specify any helpful information below)

## Tags
This Pull Request is made by @GatorEducator/team-lancaster 

@reibelj is responsible for Issue #127
@wattob and @thomad74 are responsible for Issue #125 

You can also contact me and other team members if you have any question.

### CODE REVIEW

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Please include a short summary of the proposed changes.
+
+Also include any other relevant information that is useful for this pull request.
+
+Fixes Issue #:
+
+
+## Type of change
+
+Please descirbe the pull request as one of the following:
```
Comment (Michionlion) [inline docs/pull_request_template.md]: `descirbe` -> `describe`

File: issueTemplate.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+**WARNING: Issue will be closed if it does not follow these guidelines.
```
Comment (Michionlion) [inline issueTemplate.md]: Should this line be commented out, as the other instruction are?

File: issueTemplate.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+# Descriptive Title of Issue Being Raised
+<!-- Summarize the feature, bug, or other issue in the title. -->
+# Add Detailed Comment with More Information
+<!-- Expand upon the detailed title with more in-depth information to help people understand the issue. -->
+# Select Labels Based on Issue
```
Comment (Michionlion) [inline issueTemplate.md]: Is this a necessary header? It is a step, but does this information need to be included in the issue description?

This and many other headers afterwards seem more like instructions for handling issues than headers that form a template for an issue.

File: tests/test_display.py [hunk 4]
```diff
@@ -8,3 +8,8 @@ def test_display_student_groups(capsys):
     display.display_student_groups(student_groups)
     out, err = capsys.readouterr()
     assert out.startswith('\033[0;32m' + '\033[1m' + '\033[4m' + "Group 1")
+
+
+def test_display_welcome_message():
```
Comment (Michionlion) [inline tests/test_display.py]: Why is there a test added to the PR? This should be moved to a different PR.

File: issueTemplate.md (hunk 2 above)
Comment (thomad74) [reply issueTemplate.md]: This does not necessarily need to be commented out unless you feel that it should be. I was basing this template off of another repository which had a similar 'warning' message regarding not following the guidelines. But it does not matter to me either way!

File: docs/pull_request_template.md (hunk 1 above)
Comment (reibelj) [reply docs/pull_request_template.md]: I will handle this simple fix

File: issueTemplate.md (hunk 3 above)
Comment (thomad74) [reply issueTemplate.md]: Some of them do look like instructions for handling issues, but I was basing the headers off of how I thought someone reviewing the issue would like to see them to have the most information and easiest time understanding the issue. If you feel this is unnecessary, I can edit or delete it accordingly.

File: docs/pull_request_template.md (hunk 1 above)
Comment (reibelj) [reply docs/pull_request_template.md]: I will handle this simple fix

Comment (Lancasterwu) [reply docs/pull_request_template.md]: Also, @reibelj can you change `Fixes Issue #:` to `Fixes #Issuenumber`?

Comment (reibelj) [reply docs/pull_request_template.md]: Yes I can change that as well.

File: issueTemplate.md (hunk 3 above)
Comment (Michionlion) [reply issueTemplate.md]: These issues are for you to decide how to handle. Talk with your leadership and figure out what design you want to follow! I'm just pointing out things that may perhaps be bad practice or inconsistent.

Comment (wattob) [reply issueTemplate.md]: My team has met and we think this design works. @gkapfham, what do you think about this?

Comment (wattob) [reply issueTemplate.md]: @Michionlion, My group has just updated the Issue Template to follow your suggestions. @Lancasterwu and @thomad74

**Comment by Lancasterwu** [comment]
Just make a note that this can be merged in after PR https://github.com/GatorEducator/gatorgrouper/pull/154 is merged. In order to avoid the merge conflict.
